packages/postcss-reduce-initial/data/toInitial.json

```diff
@@ -17,17 +17,19 @@
   "box-sizing": "content-box",
   "column-rule-color": "currentcolor",
   "font-synthesis": "weight style",
+  "image-orientation": "from-image",
   "mask-clip": "border-box",
   "mask-mode": "match-source",
   "mask-origin": "border-box",
-  "mask-repeat": "repeat",
+  "mask-repeat": "no-repeat",
```

Comment:
I traced the spec for this property all the way back to the original `W3C Last Call Working Draft` of 29 October 2013, and it's remained unchanged as `no-repeat` since then. The only occurrence I find of plain `repeat` initial value is in the [original working draft](https://www.w3.org/TR/2013/WD-css-masking-20130620/). So this is also a pretty old bug - which MDN didn't catch in their repo until [this merge](https://github.com/mdn/data/pull/105/files) back in 2017.

Perhaps I should amend the commit message to include a short summary of the changes?

For example:

```
* Fix incorrect initial value of `mask-repeat` and `transform-box`
* Update initial value for `image-orientation` and `ruby-position`
* Add misc. missing props that became stable / standard
```

Reply:
Looking at the git history, after @alexander-akait's tip about it, I see this one was deliberately changed in [this PR](https://github.com/cssnano/cssnano/pull/679) in 2019. Didn't occur to me to check, as I didn't think these files would be manually overridden.

The [current spec](https://www.w3.org/TR/css-masking-1/) says "no-repeat" while the [editors draft](https://drafts.fxtf.org/css-masking-1/#the-mask-repeat) says "repeat"

What's appropriate to follow here?

Reply:
I think we should base ourselves on what current browsers implement and we should avoid regressions against already filed issues. Unfortunately, which browsers cssnano supports is somewhat unclear, you can see it when people complain about things breaking in IE 11 (recently https://github.com/cssnano/cssnano/issues/1163), because some cssnano plugins use Browserslist configs to determine which transforms are allowed, others just ignore it.